deploy_nva_api.yml

```diff
@@ -198,25 +213,26 @@ Resources:
         ApplicationId: arn:aws:serverlessrepo:eu-west-1:884807050265:applications/NvaUsersAndRolesService
         SemanticVersion: 0.1.2
       Parameters:
-        ApiKeySecretArn: !Sub 'arn:aws:secretsmanager:eu-west-1:${AWS::AccountId}:secret:UserCatalogueApiKey-dIqvz5'
+        ApiKeySecretArn: !Sub 'arn:aws:secretsmanager:eu-west-1:${AWS::AccountId}:secret:UserCatalogueApiKey-p9ebkD'
```

Comment:
Will we not get in trouble when trying this template in the wrong environment?

Reply:
Overlooked, fixed by https://github.com/BIBSYSDEV/NVA-deploy/pull/13/commits/38218a987724ebf24122dbc9a05432cec489d861